docs/blog/2019-03-29-behind-the-scenes-what-makes-gatsby-great/index.md

```diff
@@ -483,11 +483,57 @@ Lighthouse is a great _first_ step to asserting whether or not your application
 
 [Webpagetest](https://webpagetest.org) is an amazing tool built by [Patrick Meenan](https://twitter.com/patmeenan). You can tweak the device used in the tests to validate that your application works even on low-performing devices that tend to be CPU-constrained. You can use a slower network to ensure that you get an accurate representation of how your users could be interacting with your application in less-than-ideal network connections. You can even choose the location of where your test is run!
 
-I liken the analogy between Lighthouse and Webpagetest as similar to the relationship between a unit test (Lighthouse) and an end to end test (Webpagetest). A unit test gives you a fair degree of confidence that what you're testing works like you expect. An end to end test gives you a near certainty that what you're testing works as expected, because you're testing in real-world conditions.
+I liken the relationship between Lighthouse and Webpagetest as similar to the relationship between a unit test (Lighthouse) and an end to end test (Webpagetest). A unit test gives you a fair degree of confidence that what you're testing works like you expect. An end to end test gives you a near certainty that what you're testing works as expected, because you're testing in real-world conditions.
 
 Both are valuable--and both have their place!
 
-<!-- TODO: finish this out and add links to CI integration -->
+### Trust, but validate
+
+As mentioned, performance is something that should be regularly monitored. Using tools like Webpagetest and Lighthouse are **great** techniques that should be used and help uncover performance problems (among other key concerns like accessibility improvements, SEO, etc.). However, the usage of these tools are oftentimes done in a reactionary, ad-hoc manner. Performing a Lighthouse test, discovering issues, and devoting a week or two to remediate is _all_ too common.
+
+Why react to something (!) with manual, ad-hoc spot checking? What if we could remediate and diagnose at pull request time?
```

Comment:
Sidenote: I miss a tutorial "how to CI with lighthouse" in documentation

Reply:
I don't think it _needs_ to be a tutorial _yet_ but I can create a stub if that's something people would like. The repository I created is a great start to getting some documentation up and created.

Reply:
maybe the stub can have a link to your repo as starting point for an author

Reply:
Sure! But this is outside the scope of this PR! Would love a PR if you're willing and able!